### PR TITLE
Update compare-products.js

### PR DESCRIPTION
**Pull request by IbrahimS2**
A needed widget where it activates the print page functionality on the compare view page

### CODE REVIEW

**Comment by IbrahimS2** [comment]
@vovayatsyuk The compare-products must be loaded so the widget can be utilized. Currently, there is a print button, but it does not do anything.

image

File: view/frontend/web/js/components/compare-products.js [hunk 1]
```diff
@@ -1,8 +1,47 @@
 (function () {
-    'use strict';
```
Comment (vovayatsyuk) [inline view/frontend/web/js/components/compare-products.js:2]: Do not replace quotes. Bring back the single quotes please.

File: view/frontend/web/js/components/compare-products.js [hunk 2]
```diff
@@ -1,8 +1,47 @@
...
+    $.widget('mage.compareList', {
+        component: 'compareList',
+
+        /** @inheritdoc */
+        _create: function () {
+            var elem = this.element,
+                products = $('thead td', elem),
+                headings;
+
+            if (products.length > this.options.productsInRow) {
```
Comment (vovayatsyuk) [inline view/frontend/web/js/components/compare-products.js:18]: Please remove everything from the `_create` function except print functionality.

**Review by vovayatsyuk — CHANGES_REQUESTED** [review]

File: view/frontend/web/js/components/compare-products.js (hunk 2 above)
Comment (IbrahimS2) [reply view/frontend/web/js/components/compare-products.js]: @vovayatsyuk done

**Comment by vovayatsyuk** [comment]
Thank you!

**Pull request merged**